docker/buildroot.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
+
+docker_image="goranche/raspbian-ua-netinst"
+
+if [ "${1}" != "IN_DOCKER" ]; then
+	docker run --privileged -ti --rm -v $(pwd):/raspbian-ua-netinst "${docker_image}"
+	exit
+fi
+
```

Comment:
😅 yes, you did 😉 

the same script is run on both the host system (as a command), as well as in the container, where it does it's magic...

it basically checks if the command line parameter is given, if not (normal condition), it will run the docker container, and inside the docker container the script will be run again, but this time with the command line parameter...

as a user, you don't need to know the parameter even exists... if you do pass it, though, it will just do a build as if it were inside the container... depending on your system, it might work, but if it's not a linux system with all dependencies installed, it will not